Tools-Override/tests.targets

```diff
@@ -214,6 +214,45 @@
       <PostExecutionTestCommandLines Include="copy /y testResults.xml %EXECUTION_DIR%\" />
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
+    </PropertyGroup>
+  
+    <!-- The results path is different on the helix machine than locally -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true' and '$(ArchiveTests)' == 'true'">
+      <_Results_Path>%USERPROFILE%\Documents\TestResults\</_Results_Path>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Runtime_Path>%RUNTIME_PATH%\UAPLayout\</_Runtime_Path>
+      <!-- Make sure the path only has one type of slashes -->
+      <_Runtime_Path>$(_Runtime_Path.Replace('/', '\'))</_Runtime_Path>
+
+      <!-- The test execution command is different for UAP. -->
+      <TestCommandLine>%RUNTIME_PATH%\Launcher\WindowsStoreAppLauncher.exe -test appxmanifest.xml $(XunitArguments) $(XunitTraitOptions)</TestCommandLine>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(BuildingUAPVertical)' == 'true'" >
+      <!-- Copy the runner files into the test directory -->
+      <RunnerFolderContents Include="$(NETCoreAppTestRootPath)\Runner\**\*" />
+
+      <TestCommandLines Include="mkdir Assets" />
+      <TestCommandLines Include="mkdir entrypoint" />
+      <TestCommandLines Include="mkdir Properties" />
+      <TestCommandLines Include="mkdir WinMetadata" />
+      <TestCommandLines Include="mklink /H %(RunnerFolderContents.RecursiveDir)%(RunnerFolderContents.Filename)%(RunnerFolderContents.Extension) %RUNTIME_PATH%\Runner\%(RunnerFolderContents.RecursiveDir)%(RunnerFolderContents.Filename)%(RunnerFolderContents.Extension)" />
+
+      <!-- Copy the runtime binaries over -->
+      <RuntimePathContents Include="$(RuntimePath)\**\*" />
+      <TestCommandLines Include="mklink /H %(RuntimePathContents.RecursiveDir)%(RuntimePathContents.Filename)%(RuntimePathContents.Extension) $(_Runtime_Path)%(RuntimePathContents.Filename)%(RuntimePathContents.Extension)" />
+
+      <!-- Copy the log files and the results files from the Documents folder to the test folder -->
+      <PostExecutionTestCommandLines Include="move $(_Results_Path)$(XunitTestAssembly).xml .\$(XunitResultsFileName)" /> 
+      <PostExecutionTestCommandLines Include="move $(_Results_Path)$(XunitTestAssembly).txt .\logs.txt" />
+      <PostExecutionTestCommandLines Include="type logs.txt" />
```

Comment:
Because we sadly don't have a good way to redirect the output. We dump the logs into logs.txt and then, to see that on the console, we do a type logs.txt.

If you can think of a better approach, I would be very happy to use that instead! :D